apps/ledger-live-mobile/src/components/TransactionsPendingConfirmationWarning.tsx

```diff
@@ -53,9 +47,27 @@ const TransactionsPendingConfirmationWarning = ({
         </LText>
       </BottomModal>
     </View>
+  );
+};
+
+export const TransactionsPendingConfirmationWarningForAccount = ({
+  maybeAccount,
+}: {
+  maybeAccount: AccountLike;
+}) => {
+  return isAccountBalanceUnconfirmed(maybeAccount) ? (
+    <TransactionsPendingConfirmationWarningContent />
   ) : null;
 };
 
+export const TransactionsPendingConfirmationWarningAllAccounts = () => {
+  const areSomeAccountsBalanceUnconfirmed = useSelector(
+    areSomeAccountsBalanceUnconfirmedSelector,
+  );
+  if (!areSomeAccountsBalanceUnconfirmed) return null;
+  return <TransactionsPendingConfirmationWarningContent />;
+};
+
```

Comment:
Here we split this component in 2 for 2 types of consumer:
- one that wants to use all accounts and thus needs a component that auto updates only when these accounts have/don't have some transactions pending, thus it's optimized to only re-render when that changes because it uses a selector made for that.
- one for a unique account